src/geotools/src/main/java/org/locationtech/geogig/geotools/plumbing/DataStoreImportOp.java

```diff
@@ -0,0 +1,165 @@
+/* Copyright (c) 2016 Boundless and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/org/documents/edl-v10.html
+ *
+ * Contributors:
+ * Erik Merkle (Boundless) - initial implementation
+ */
+package org.locationtech.geogig.geotools.plumbing;
+
+import org.geotools.data.DataStore;
+import org.locationtech.geogig.api.AbstractGeoGigOp;
+import org.locationtech.geogig.api.RevCommit;
+import org.locationtech.geogig.api.RevTree;
+import org.locationtech.geogig.api.porcelain.AddOp;
+import org.locationtech.geogig.api.porcelain.CommitOp;
+import org.locationtech.geogig.repository.WorkingTree;
+
+import com.google.common.base.Supplier;
+
+/**
+ * Imports feature tress (layers) into a repository from a GeoTools {@link DataStore}.
+ *
+ */
+public class DataStoreImportOp extends AbstractGeoGigOp<RevCommit> {
```

Comment:
At a first glance it's not clear what's the difference with `ImportOp`. Maybe add some documentation it decorates it by adding a commit on everything that's been imported?